modules/gdscript/gdscript_editor.cpp

```diff
@@ -1157,9 +1157,20 @@ static void _find_identifiers_in_base(const GDScriptCompletionIdentifier &p_base
 						tmp.get_property_list(&members);
 					}
 
+					static HashMap<String, String> theme_color_name_map;
+					if (theme_color_name_map.is_empty()) {
+						theme_color_name_map.insert("x", "axis_x_color");
+						theme_color_name_map.insert("y", "axis_y_color");
+						theme_color_name_map.insert("z", "axis_z_color");
+						theme_color_name_map.insert("w", "axis_w_color");
```

Comment:
There is out to be a better place for this, maybe a static class member that's initialized in some dedicated place. But it's fine as-is for now.

Reply:
Done, I think.